packages/blockchain-wallet-v4-frontend/src/components/Alerts/messages.tsx

```diff
@@ -1159,11 +1159,18 @@ export const getAlertContent = (message, data = undefined) => {
       )
     case C.LOCKED_WITHDRAW_ERROR:
       return buildMessageTemplate(
-        <FormattedMessage
-          id='copy.error.locked_withdraw_error'
-          defaultMessage='Your crypto will be available to be withdrawn within {days} days.'
-          values={data}
-        />
+        data === 0 || data === 1 ? (
```

Comment:
sometimes API will return lock time in minutes or hours which leads us in moment rounding to 0 days, in such case we want to show 1 day